doc/source/ray-core/ray-dag.rst

```diff
@@ -0,0 +1,155 @@
+.. _ray-dag-guide:
+
+Building Computation Graphs with Ray DAG API
+============================================
+
+With ``ray.remote`` you have the flexibility of running an application where
+computation is executed remotely at runtime. For a ``ray.remote`` decorated
+class or function, you can also use ``.bind`` on the body to build a static
+computation graph.
+
+.. note::
+
+     Ray DAG is designed to be developer facing API where recommended use cases
```

Comment:
Do we also '@DeveloperAPI' the APIs?

Reply:
i can do it in the same PR if that makes more sense